doc/tasks/readme.md

```diff
@@ -0,0 +1,737 @@
+GUI-Dialog geöffnet wird, der den Text der Schriftrolle enthält.
+
+### Eine Entität im Dungeon "inspizieren"
+
+Entitäten im Dungeon können vom Spielcharakter inspiziert werden, wodurch sich ein GUI-Dialog öffnet, der Informationen
+über die inspizierte Entität anzeigt.
+
+### Ein Monster angreifen
+
+Der Spielcharakter kann mit einer ausgerüsteten Waffe ein Monster angreifen und dessen Gesundheit Schaden zufügen.
+
+### Schaden von einem Monster zugefügt bekommen
+
+Der Gesundheit des Spielcharakters kann durch ein Monster Schaden zugefügt werden.
+
+### Monster besiegen
+
+Wenn die Gesundheit eines Monsters auf 0 fällt, wird es besiegt und stirbt, wodurch es ggfs. Erfahrungspunkte und
+Items aus seinem Inventar (falls vorhanden) fallen lässt.
+
+### Einen Block verschieben
+
+Per Interaktion kann der Spielcharakter einen Block im Dungeon in eine bestimmte Richtung verschieben.
+
+### Einen Schalter (im Dungeon) betätigen
+
+Per Interaktion kann der Spielcharakter den Zustand eines Schalters im Dungeon ändern. Welche Zustände es Schalter
+annehmen kann, hängt von der Art des Schalters ab.
+
+### Einen Crafting-Schritt durchführen
+
+Bei geöffnetem Inventar eines Craftingcontainers können Items aus dem Container-Inventar ausgewählt werden, auf die
+ein auswählbares Craftingrezept (also eine Transmutationsregel) angewandt wird, um die Menge der ausgewählten Items zu
+verändern. Nach der Auswahl der Items und des Craftingrezepts muss der Crafting-Schritt per GUI-Button bestätigt werden.
+
+## Spielszenarien
+
+### NPC stellt Frage
+
+Im Level steht ein Questgeber, mit dem der Spieler reden kann. Im Gespräch stellt der NPC
+dem Spieler eine Frage, die er beantworten muss. Beantwortet der Spieler die Frage richtig,
+bekommt er eine Belohnung. Beantwortet der Spieler die Frage falsch, bekommt er keine
+Belohnung.
+
+### Monster greift mit Frage an
+
+Im Level läuft ein Monster herum. Wenn das Monster den Spieler sieht, läuft es in den
+Spieler hinein (bis zur Kollision). Bei Kollision stellt das Monster dem Spieler eine Frage.
+Beantwortet der Spieler die Frage richtig, stirbt das Monster. Beantwortet der Spieler die
+Frage falsch, bekommt er viel Schaden und das Monster rennt weg.
+
+### Truhe ist mit Frage verschlossen
+
+Im Level steht eine Schatzkiste herum. Wenn der Spieler die Schatzkiste öffnen will, stellt
+diese ihm eine Frage. Beantwortet der Spieler die Frage richtig, geht die Schatzkiste auf.
+Beantwortet der Spieler die Frage falsch, bleibt die Schatzkiste verschlossen.
+
+### Mehrere Truhen aber nicht alle sind richtig
+
+Der Spieler kommt in einen Raum. Der Questgeber stellt ihm eine Frage. Die
+Antwortmöglichkeiten sind in Form von Schatzkisten im Raum verteilt. Wenn der Spieler mit
+den Schatzkisten interagiert, sagen diese ihm, welche Antwort sie sind. Der Spieler muss die
+Schatzkiste/n mit der/den richtigen Antwort/en öffnen.
```

Comment:
Das passt für mich nicht in die üblichen Spielmechaniken im Dungeon.

Der Spieler interagiert mit den Truhen, öffnet sie und schaut sich den Inhalt an. Er muss die richtigen Items (aus den richtigen Truhen) aufsammeln (entsprechen der Antwort oder sind mit der Antwort gelabelt) und dem Questgeber bringen.